### PR TITLE
Release notes for 0.14.1

### PR DESCRIPTION
**Pull request by jni**
If I can get one or two people to look these over, I'll merge, update the version number, and create the tag.

The first two commits are updates to the 0.14.0 release notes that never made it to v0.14.x from master.

Thanks!

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Do you want things like: 
https://github.com/scikit-image/scikit-image/pull/3381
https://github.com/scikit-image/scikit-image/pull/3236
to be made into the changelog

I think these need to make it to the changelog
https://github.com/scikit-image/scikit-image/pull/3338
https://github.com/scikit-image/scikit-image/pull/3280
https://github.com/scikit-image/scikit-image/pull/3295
https://github.com/scikit-image/scikit-image/pull/3292

File: doc/release/release_0.14.rst [hunk 1]
```diff
@@ -1,3 +1,109 @@
...
+  for reporting this bug and for their patient help with this fix)
+- Fix import of ``moments_coord_central`` in ``measure`` package, which meant
+  that neither it nor ``moments_normalized`` could be correctly imported from
+  the ``measure`` namespace (#3374)
+
+Enhancements
+------------
+- "Reflect" mode in transforms now works fine when an image dimension has size
+  1 (#3179)
+- ``img_as_float`` now allows single-precision (32-bit) float arrays to pass
```
Comment (hmaarrfk) [inline doc/release/release_0.14.rst]: So useful. see requested changes in comment.

**Comment by jni** [comment]
@hmaarrfk thanks for the very useful review! I've added all the PRs except #3381, which is less necessary now. How did you pick the PRs that I'd missed?

**Comment by hmaarrfk** [comment]
I went through and clicks on all the backports. Very tedious i know...

dont know a good way to do it for .15

**Comment by hmaarrfk** [comment]
FWIW, I disagree with you skipping on the mention of your own PR for sphinx. That kind of thing is what allows us to accept other's contributions with confidence.

**Comment by jni** [comment]
@hmaarrfk 

> FWIW, I disagree with you skipping on the mention of your own PR for sphinx. That kind of thing is what allows us to accept other's contributions with confidence.

What do you mean? Of course the PR is important in itself, but now that 1.7.9 is out, backporting and highlighting it in the release notes feels like unnecessary noise, especially since the contributors and reviewers involved have been credited in many other PRs. It will also be included in 0.15.

**Comment by hmaarrfk** [comment]
@jni I swear i found it backported. Oh well. not too important.

**Pull request merged**